CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# dbt_stripe_source v0.13.0
+[PR [#87](https://github.com/fivetran/dbt_stripe_source/pull/87)] includes the following updates:
+
+## Breaking Change
+- This package assumes that amount-based fields, which as raw values are represented in the smallest denomination in Stripe, are cent-based. This PR shifts the existing conversion from cents to dollars to further upstream. Previously, currency-related fields were converted in downstream models, but now have been converted directly in staging models. Since currency-related fields now have different values, this is a breaking change. 
```

Comment:
Can you highlight how this differs from how this package treated this previously.

Reply:
updated